Detectors/MUON/MID/Raw/include/MIDRaw/LocalBoardRO.h

```diff
@@ -17,6 +17,7 @@
 
 #include <cstdint>
 #include <array>
+#include <ostream>
```

Comment:
Can you use `<iosfwd>`? Including ostream in a header file might bring in a lot of crap which is then instanciated everywhere resulting in longer compilation / linking times.